ws-nextjs-app/pages/[service]/live/[id]/live.test.tsx

```diff
@@ -125,6 +125,27 @@ describe('Live Page', () => {
     },
   );
 
+  it.each`
```

Comment:
Is this test a bit overkill? Since the logic for seoTitle/title is already being checked by another test.

An alternative would be to simply check the headline field exists in  the scriptTags. (This could be expanded as we add more fields)

Or should we not check? I don't think we do this for the articlePages.

Reply:
I think these are fine to have. They are likely things that, once set, will never change or need updating. They are readable to me and make sense. If we find that they are harder to unpick / extend then we can tackle that work then I feel.